sdk/src/services/vault/client.rs

```diff
@@ -57,8 +57,8 @@ impl Default for VaultClient {
 }
 
 impl VaultClient {
-    pub const FETCH_SECRETS: &str = "/v1/secret/data";
-    pub const UPDATE_SECRET: &str = "/v1/secret/data";
+    pub const FETCH_SECRETS: &'static str = "/v1/secret/data";
```

Comment:
New rust version enforce this

Ref issue: https://github.com/rust-lang/rust/issues/115010